project/Build.scala

```diff
@@ -903,4 +905,18 @@ object DottyInjectedPlugin extends AutoPlugin {
       }
     ))
   }
+
+  private def projectChecks(): Unit = {
+    val scalaScala = new File("scala-scala")
+    if (!scalaScala.exists()) {
+      println(
+        s"""[WARNING] Missing `dotty/scala-scala` library
+           |You can clone the library with:
+           |  > git clone https://github.com/DarkDimius/scala.git ${scalaScala.getAbsolutePath}
```

Comment:
You can do it in one line with git clone -b, see http://dotty.epfl.ch/docs/contributing/workflow.html